### ISSUE TITLE
Tm Theme Editor

### ISSUE THREAD

**Issue by earlev4** [ericwbailey/a11y-syntax-highlighting#1]
Hi!

Good day to you! I greatly appreciate the efforts and work going into the a11y-syntax-highlighting project. I have seen much positive feedback on A11y Dark theme on Carbon and I share that sentiment. A11y Dark is one of the best themes I have seen in a while. Are you aware of any efforts to add A11y Dark theme to the Tm Theme Editor Website? 

https://tmtheme-editor.herokuapp.com/#!/editor/theme/Monokai

I would be happy to assist to build one but would need to know all of the possible scope names and associated foreground/background hex values. I would not want to make any assumptions and diverge from the appropriate color theme.

Thanks so much!

**Comment by ericwbailey** [comment]
Hey there! Thank you for the kind words, it means a lot to me.

I've got [color names and values listed in the README](https://github.com/ericwbailey/a11y-syntax-highlighting#a11y-dark) for the dark and light themes, as well as their corresponding WCAG level of compliance when placed on the background color.

If you want to create a PR for tmTheme Editor, I say go for it! I'd appreciate a link when you're done, just to check it out 🙂

**Comment by alamodey** [comment]
can you make a theme for Xcode as well?

**Comment by ericwbailey** [comment]
@alamodey Great idea! I'll start chipping away at it.

**Comment by ericwbailey** [comment]
@alamodey I added a dark mode theme for Xcode: https://github.com/ericwbailey/a11y-syntax-highlighting/tree/main/dist/xcode

I don't spend a lot of time in that editor. Does it feel right to you? If so, I'll whip up a light theme. Happy to make tweaks, too.

**Comment by alamodey** [comment]
<img width="664" alt="Screen Shot 2020-12-11 at 5 18 13 pm" src="https://user-images.githubusercontent.com/55496019/101870028-eb329b80-3bd4-11eb-9510-e680ce4d580d.png">
It looks a bit different to what I expected

**Comment by ericwbailey** [comment]
Are you sure the correct theme is highlighted? This is how the preview looks on my end of things:

<img width="942" alt="The 'A11Y Dark' theme selected in Xcode, with exampled colored code tokens." src="https://user-images.githubusercontent.com/634191/101937348-bf50ef00-3baf-11eb-9bfb-b125845d7dd7.png">

**Comment by alamodey** [comment]
<img width="868" alt="Screen Shot 2020-12-12 at 10 02 03 am" src="https://user-images.githubusercontent.com/55496019/101962781-27abd900-3c61-11eb-8024-70d794c97111.png">

**Comment by alamodey** [comment]
Is it because I'm in dark mode?

**Comment by ericwbailey** [comment]
Oh, I think I got it! 

Set your colors to the following:

- Background: `#2B2B2B`
- Current Line: `#000000`
- Selection: `#000000`
- Cursor: `#FFFFFF`
- Invisibles: `#565655`

I assumed these values were included as part of the theme data, but it appears that they're not?

**Comment by alamodey** [comment]
There must be a way to include it in the theme data because all the other themes I download on Xcode work fine.

**Comment by ericwbailey** [comment]
Yeah, I'll investigate. So weird.

**Comment by ericwbailey** [comment]
So, I tried toggling dark mode on and off and the background, current line color, etc. persist. I'm also not seeing an option to include those values in the export, as exporting is just copying the `.xccolortheme` out of `~/Library/Developer/Xcode/UserData/FontAndColorThemes/`. 

Do you know of another way to export?

**Comment by alamodey** [comment]
I am not familiar with creating themes but what if you went to the github of another theme like this: https://github.com/bojan/xcode-one-dark

and just replace their colours with yours?

**Comment by ericwbailey** [comment]
I just ran through and manually compared those `key` definitions with what I have. Weirdly, I have more, but nothing in either theme that speaks to the missing background color, etc.
